tests/h/services/nipsa_test.py

```diff
@@ -3,73 +3,57 @@
 from h.services.nipsa import NipsaService, nipsa_factory
 
 
-@pytest.mark.usefixtures("users", "reindex_user_annotations")
+@pytest.mark.usefixtures("users")
 class TestNipsaService:
-    def test_fetch_all_flagged_userids_returns_set_of_userids(self, db_session):
-        svc = NipsaService(db_session)
-
+    def test_fetch_all_flagged_userids_returns_set_of_userids(self, svc):
         assert svc.fetch_all_flagged_userids() == {
             "acct:renata@example.com",
             "acct:cecilia@example.com",
         }
 
-    def test_is_flagged_returns_true_for_flagged_users(self, db_session, users):
-        svc = NipsaService(db_session)
-
+    def test_is_flagged_returns_true_for_flagged_users(self, svc, users):
         assert svc.is_flagged("acct:renata@example.com")
         assert svc.is_flagged("acct:cecilia@example.com")
```

Comment:
Super nit pick, but why prove this twice? I think the second user is there for the `fetch_all_flagged_userids` later.
```suggestion
```